tools/travis.ps1

```diff
@@ -42,7 +42,7 @@ function Set-DailyBuildBadge
     $storageAccountKey = $Env:TestResultAccountKey
 
     # this is the url referenced in README.MD which displays the badge
-    $platform = if ( $IsOSX ) { "OSX" } else { "Linux" }
+    $platform = if ( $IsLinux ) { "Linux" } else { "OSX" }
```

Comment:
Minor comment - maybe replace "OSX" too?

Reply:
The file stored in Azure for the badge has OSX in the filename.  I'm fine leaving that one.